apps/camera/js/views/preview-gallery.js

```diff
@@ -134,6 +134,10 @@ return View.extend({
     if (event.deltaMode !== event.DOM_DELTA_PAGE || event.deltaY) {
       return;
     }
+    // First, stop the video if there is one and it is playing.
+    if (this.videoPlaying) {
+      this.onVideoPaused();
```

Comment:
Note that the onVideoPaused() function does not appear to actually stop the video, so the comment here is misleading.

Also, the code that registers this onFrameWheel event handler says that it is going to be handling arrow up and down keys. But it is a mouse wheel event handler instead, so there is something wrong with the comment at line 77.

Finally, should this code look at the magnitude of deltaX before deciding whether to emit the swipe gesture? Or does the fact that this is in page mode mean that it will take a substantial swipe motion to trigger this event?